README.developer.md

```diff
@@ -104,16 +108,29 @@ To install it into `builddir/bin` use:
 meson install -C builddir --destdir bin
 ```
 
-After building, three binaries are available:
+After building, four binaries are available:
```

Comment:
Wouldn't it be easier for future to use *following* instead of an exact number?

Reply:
Changed.